openapi/paths/tags-configurations@{id}.yaml

```
@@ -0,0 +1,80 @@
+parameters:
+  - $ref: ../components/parameters/resourceId.yaml
+get:
+  x-products:
+    - Core
+  tags:
+    - Tags
+  summary: Retrieve a tags configuration
+  operationId: GetTagsConfiguration
+  x-sdk-operation-name: get
+  description: Retrieve a tags configuration with specified identifier string.
```

nitpick: based on our writing style
```suggestion
  description: Retrieve a tags configuration with a specified ID.
```